catalog-info.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
     github.com/team-slug: elastic/kibana-tech-leads
     buildkite.com/project-slug: elastic/kibana
     sonarqube.org/project-key: elastic_kibana_AYvOkAHeQZlFqhqWIr9
+    pagerduty.com/service-id: P3CJ6U9
```

Comment:
I think we want the same serverless rotation `PC4TBJY` for this one as well

Reply:
`PC4TBJY` is kibana-serverless in PD, whereas `P3CJ6U9` is kibana-sdh.